public/app/features/explore/AddToDashboard/AddToDashboardModal.tsx

```diff
@@ -39,11 +39,13 @@ export const AddToDashboardModal = ({ onClose, queries, visualization, onSave }:
 
     if (error) {
       switch (error.status) {
-        case 'name-exists':
         case 'empty-name':
```

Comment:
Can we add another error for this case specifically? "Name is required."

Reply:
that's already handled:

<img width="375" alt="Screenshot 2022-03-03 at 12 08 47" src="https://user-images.githubusercontent.com/1170767/156562002-0c79c567-5cce-49ad-b9a3-852ea6db1bcb.png">

Reply:
actually, this can also generate this error (for instance if you put in a whitespace and try to save).

<img width="297" alt="Screenshot 2022-03-03 at 12 14 58" src="https://user-images.githubusercontent.com/1170767/156562904-d5e1a15e-1d2f-41e3-a204-51b293ebadb1.png">

i think in both cases "Dashboard name cannot be empty." would be a good option, wdyt?

Reply:
I would phrase it in a positive actionable way like "Dashboard name is required." or "Please enter a dashboard name."

Reply:
<img width="853" alt="Screenshot 2022-03-03 at 12 52 21" src="https://user-images.githubusercontent.com/1170767/156568396-45ae8c0d-5066-43cf-a843-47d6b6e597ab.png">